scripts/goog_module/convert-file.sh

```diff
@@ -201,12 +215,11 @@ step3() {
   done
 
   local missing_requires=$(perl -nle'print $& while m{(?<!'\'')Blockly(\.\w+)+}g' "${filepath}")
-  local missing_require_lines=$(echo "${missing_requires}" | wc -l)
-  if [[ "${missing_require_lines}" -gt "0" ]]; then
+  if [[ ! -z "${missing_require_lines}" ]]; then
     err "Missing requires for: ${missing_requires} Please manually fix."
   fi
 
-  inf "Add missing nullability modifier..."
+  inf "Add missing nullability modifiers..."
   perl -pi -e 's/@(param|return) \{([A-Z])/@\1 \{?\2/g' "${filepath}"
```

Comment:
I think this might not work because there are a lot of `Function` types which are considered primitives and don't need a modifier. And we decided not to do it as we go anyway so you could probably take this part out rather than solve it. I've noticed some cases where there is no modifier so by default it's nullable but logically they should probably not be nullable. Those cases will be easier to notice if we haven't mechanically added the `?` without checking each case by hand

Reply:
I can remove it.